packages/rangelink-vscode-extension/scripts/rotate-release-board.sh

```
@@ -0,0 +1,283 @@
+            nodes {
+              ... on ProjectV2ItemFieldSingleSelectValue {
+                name
+                field { name }
+              }
+            }
+          }
+        }
+      }
+    }
+  }
+}'
```

Shellcheck (0.11.0)

[info] Expressions don't expand in single quotes, use double quotes for that.

(SC2016)